integration/commands_administration_test.go

```diff
@@ -853,3 +853,28 @@ func TestCommandsAdministrationWhatsMyURI(t *testing.T) {
 	require.Equal(t, 2, len(ports))
 	assert.NotEqual(t, ports[0], ports[1])
 }
+
+func TestCommandsAdministrationListDatabases(t *testing.T) {
```

Comment:
Please move it after `TestCommandsAdministrationCreateDropListDatabases`

Reply:
Done, plus changed from `SkipForTigris` to `SkipForTigrisWithReason`.